app/models/user.rb

```diff
@@ -16,4 +17,11 @@ class User < ApplicationRecord
     length: {maximum: Settings.maximum_length_phone}
 
   scope :sort_by_name, ->{order :name}
+  USER_PARAMS = %i(:name email password password_confirmation role).freeze
```

Comment:
```suggestion
  USER_PARAMS = %i(name email password password_confirmation role).freeze
```
Hằng số cho lên trên đầu class đi em